stdlib/csv/0/csv.rbs

```diff
@@ -3002,7 +3002,7 @@ end
 #     table['Name'] # => ["Foo", "Bar", "Baz"]
 #
 class CSV::Table[out Elem] < Object
-  include Enumerable[untyped]
+  include Enumerable[Elem]
```

Comment:
Is this change correct? If my understanding is correct, `Elem` means a row of the table. So `#each` and Enumerable should use `Elem`, right?